python/nav/util.py

```diff
@@ -30,6 +30,11 @@
 
 import IPy
 
+try:
+    from importlib.resources import files, as_file
+except ImportError:  # Python 3.7!
+    from importlib_resources import files, as_file
+
```

Comment:
I find it slightly confusing that other NAV modules are now importing `files` directly from `nav.util`, as the name is quite generic.  Could we perhaps rename the import to `package_files` or something similar?

Reply:
I've renamed it to `resource_files` so it looks similar to `resource_bytes`. When on python 3.9 we could `from importlib import resources` and rename things to `resources.files`.